docs/modules/user-guide/partials/proc_adding-commands-to-a-devfile.adoc

```diff
@@ -88,7 +88,7 @@ projects:
 components:
   - name: maven
     container:
-      image: eclipe/maven-jdk8:latest
+      image: eclipse/maven-jdk8:latest
```

Comment:
We should consider including a registry here for clarity

Reply:
@amisevsk Do you mean to make it clear that it registry can be specified here as well, or that eclipse che images are on quay.io?

Reply:
Both I suppose. I think we're generally trying to move away from the implicit `docker.io` when there's no registry, but I don't actually know where `eclipse/maven-jdk8` is stored.

Reply:
I was not able to find maven-jdk8, I think we can replace it with `quay.io/eclipse/che-java11-maven:nightly`